bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/XmlNonElementItem.java

```diff
@@ -141,6 +144,9 @@ public BXml descendants() {
     public XmlValue getItem(int index) {
         if (index == 0) {
```

Comment:
Shall we do the `index > 0` here similar to in `XmlItem` to keep the functions consistent?

Reply:
It will be more readable to arrange it like this,
```
if (index = 0) {
    return ...;
}
if (index > 0) {
    return ...;
}
throw Error...;
```